### DISCUSSION TITLE
Any way to reduce borders between elements?

### DISCUSSION THREAD

**Discussion by kingofdunghill** [darkreader/darkreader#9588] [Q&A] [answered]
Great tool, but unfortunately the many borders between elements bother me a bit (for example the grey lines, which were inserted on github nearly everywhere). Is it possible to maybe reduce the contrast in the color a bit or somehow disable them completely?

Thanks

**Comment by Gusted** [comment, marked as answer]
It's not possible to reduce/remove the borders without affecting anything else on the page. As for disabling, that will look bad:

![image](https://user-images.githubusercontent.com/25481501/185285223-10f8d9e5-74ab-4f47-9963-e801abe3ebce.png)

And reducing the contrast or colors from just one of the elements will make it look odd and wouldn't comply with the overall color scheme that is being generated.
